bin/ruby-install

```diff
@@ -18,6 +18,11 @@ fi
 
 init || exit $?
 
+if [[ ! -w "$install_dir" ]]; then
+	log "seems like you need necessary write permission on $install_dir"
```

Comment:
Error message should be more authoritative. `"installation directory is not writable by the user: $install_dir"`

Reply:
I see

Reply:
Also use `error "..."` or `fail "..."` (which also does a `exit -1`) instead of `log`, as this is an error message.